string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,13 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  var firstLetter = word[0];
+  var lowerCase = word.slice(1);
+  if (word.toLowerCase() === 'javascript') {
+    return 'JavaScript';
+  } else {
```

Comment:
You should make this a standalone `if` statement before the normal work of your function. This is called a "guard", it is a special condition you want to check for before the regular work for the function executes. By doing so, you can deal with special cases first, the just do the normal work of your function. Here is this code properly restructured to use a guard, as an example:

```
function capitalizeWord(word) {
  if (word.toLowerCase() === 'javascript') {
    return 'JavaScript';
  }
  var firstLetter = word[0].toUpperCase();
  var lowerCase = word.slice(1).toLowerCase();
  return firstLetter + lowerCase;
}
```

I made a few optimizations to your code too, take a look at those for some tips going forward 👀